pkg/block/gs/main_test.go

```diff
@@ -69,12 +69,26 @@ func TestMain(m *testing.M) {
 	}
 
 	// Create the test client and bucket
-	blockURL := fmt.Sprintf("http://%s/storage/v1/", endpoint)
+	blockURL := fmt.Sprintf("http://%s/storage/v1/", url.PathEscape(endpoint))
 	client, err = storage.NewClient(ctx, option.WithEndpoint(blockURL), option.WithoutAuthentication())
 	if err != nil {
 		log.Fatalf("Could not create gs client: %s", err)
 	}
 
+	// Wait for the container to be ready by listing buckets
+	if err := pool.Retry(func() error {
```

Comment:
Can we please explicitly define the max wait?

Reply:
The default is 1 minute - which is more than enough for this issue.

Reply:
I'm sure it's more than enough - I think that for visibility sake, it makes sense to explicitly define it in dockerpool setup

Reply:
Less code == less bugs

Reply:
Very well - not a hill I'm will to die on